### PR TITLE
Correct drops

### PR DESCRIPTION
**Pull request by erkam2002**

### CODE REVIEW

File: src/pocketmine/entity/Sheep.php [hunk 1]
```diff
@@ -123,8 +124,9 @@ public function spawnTo(Player $player){
 	 */
 	public function getDrops(){
 		$drops = [
-			ItemItem::get(ItemItem::WOOL, $this->getColor(), 1)
+			ItemItem::get(ItemItem::WOOL, $this->getColor(), 1),
+            ItemItem::get(ItemItem::RAW_MUTTON, 0, rand(1, 2))
```
Comment (Leo3418) [inline src/pocketmine/entity/Sheep.php]: Wrong indentation. Please fix this.

Comment (Leo3418) [inline src/pocketmine/entity/Sheep.php]: Wrong indentation. Please fix this.

Comment (erkam2002) [inline src/pocketmine/entity/Sheep.php]: Fixed

**Comment by erkam2002** [comment]
But i added just more drops i dont changed anthink other you can add this later..

**Comment by MasterOfTheRealm** [comment]
It's just few more lines of code

**Comment by xISRAPILx** [comment]
@erkam2002 I can create an PR in your path1

**Comment by Leo3418** [comment]
I have made necessary modifications as in 179254d.

**Comment by erkam2002** [comment]
@Leo3418 not my problem i tested it with my first code before adding loot etc..

**Comment by Leo3418** [comment]
Yes, it's still your responsibility. You should test **every** commit you made before pushing them.

**Pull request merged**